tdrive/backend/node/src/services/webdav/services/index.ts

```
@@ -0,0 +1,36 @@
+import gr from "../../global-resolver";
+
+import { Consumes, logger } from "../../../core/platform/framework";
+// import { FilesService } from "src/services/files";
+import { FileServiceImpl } from "src/services/files/services";
+
+
+@Consumes(["files"])
+export class WebDAVServiceImpl {
+    version: "1";
+    filesService: FileServiceImpl;
+
+    async init(): Promise<this> {
+        // TODO: check that only neccessary is included 
+        // TODO: properly declare all the functions
+        try {
+            logger.debug('WebDAVServiceImpl::init() -> is called');
+            
+            // await Promise.all([(this.filesService = gr.services.files.context.getProvider<FileServiceImpl>("files"))]);
+        } catch (err) {
+            logger.error("Error while initializing webdav service", err);
+        }
```

we really need to stop just ignoring exceptions. If it's an exception we expect and know how to handle, sure, but otherwise...